src/Sulu/Component/Content/Document/Behavior/PageBehavior.php

```diff
@@ -2,8 +2,6 @@
 
 namespace Sulu\Component\Content\Document\Behavior;
 
-interface PageBehavior extends 
-    WebspaceBehavior,
-    
+interface PageBehavior extends WebspaceBehavior
```

Comment:
@dantleech Fixed the syntax error, but still not really sure if we should keep this behavior... /cc @chirimoya